CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+We would ❤️ for you to contribute to Appwrite and help make it better! As a contributor, here are the guidelines we would like you to follow:
+
+## Code of Conduct
+
+Help us keep Appwrite open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+## Getting Started
+
+Install using composer:
+```bash
+composer require appwrite/php-runtimes
+```
+
+```php
+<?php
+
+require_once '../vendor/autoload.php';
+
+use Appwrite\Runtimes\Runtimes;
+
+$runtimes = new Runtime();
+
+// returns all supported cloud function runtimes
+Runtimes::getAll();
+```
+
+## Adding a new Runtime
+
+After the Docker image is created, this must be added to the main class of this library.
+
+References to this must be added to the constructor of `src/Runtimes/Runtimes.php`.
+
+Example:
+
+```php
+$dotnet = new Runtime('dotnet', '.NET');
+$dotnet->addVersion('5.0', 'mcr.microsoft.com/dotnet/runtime:5.0-alpine', 'appwrite/env-dotnet-5.0:1.0.0', [System::X86, System::ARM]);
+$runtimes[] = $dotnet;
+```
+
+## Function Runtimes Checklist
+
+The following checklist aims to ensure that a function runtime gets added successfully
+
+- [ ] Create a PR to add the runtime to [open-runtimes/open-runtimes](https://github.com/open-runtimes/open-runtimes)
```

Comment:
I'm not aware of the steps required, but I feel we miss a lot of steps here. This should be a step by step guide, allowing devs with zero knowledge about the project to be able to contribute.

Reply:
Should we incorporate all the steps from Open Runtimes' Contributing Guide here as well then?